src/runtime_src/core/common/utils.h

```diff
@@ -34,8 +34,11 @@
 #define CU_AP_CONTINUE	(0x1 << 4)
 #define CU_AP_RESET	(0x1 << 5)
 
+std::string convert_version(std::stringstream& version);
```

Comment:
Remove.  This function is not used outside of utils.cpp.

Reply:
This  function is  called in  src/runtime_src/core/common/info_platform.cpp  .  the above  header file is  included in src/runtime_src/core/common/info_platform.cpp .  So  this  declaration is needed